### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by HadarKaslasi** [ContinualAI/avalanche#1317] [Q&A] [answered]
Hello, and thanks for the promising effort with avalanche rl.

I've tried installing the package using :
pip install git+https://github.com/ContinualAI/avalanche-rl.git

However, the result is 2 seperate folders, where needed files are split (I'm assuming) between the two. 

![image](https://user-images.githubusercontent.com/81651005/224571388-c8488639-6716-4ae3-acb0-43c20643d0cd.png)

For example, when running the import command:
from avalanche.benchmarks.generators.rl_benchmark_generators import gym_benchmark_generator

the error is:
ModuleNotFoundError: No module named 'avalanche.benchmarks.generators.rl_benchmark_generators'

which is true because the needed file is in the 'avalanche_rl' folder and not the 'avalanche' folder.

Installing with pip install avalanche-lib Doesn’t solve the problem.

Does anyone have a clue?
Thanks, Hadar

**Comment by AntonioCarta** [comment, marked as answer]
Hi @HadarKaslasi. We are currently updating avalanche-rl. We made a lot of changes in avalanche to improve support for avalanche-rl. You can follow the work [here](https://github.com/ContinualAI/avalanche-rl/pull/12)

**Comment by HadarKaslasi** [comment]
Hi - Thanks!! ( so, should I try installing again once discussion in the
link will be resolved? - sorry - new to the field:) )
Hadar

‫בתאריך יום ב׳, 13 במרץ 2023 ב-12:22 מאת ‪Antonio Carta‬‏ <‪
***@***.***‬‏>:‬

**Reply by AntonioCarta** [reply]
Hi @HadarKaslasi, the master branch of avalanche-rl should be fixed now. If you still have problems you can open an issue in the avalanche-rl repository (it's easier to track if it's there).
